leptonic/Cargo.toml

```diff
@@ -30,33 +30,33 @@ js-sys = "0.3.68"
 leptos = "0.6.5"
 leptos_reactive = "0.6.5"
 leptos-tiptap = { version = "0.7.0", optional = true }
-leptos-use = { version = "0.10.2", features = ["math"] }
+leptos-use = { version = "0.12.0", features = ["math"] }
 icondata = { version = "0.3.0" }
 leptos_meta = { version = "0.6.5", features = [] }
 leptos_router = "0.6.5"
 serde = "1.0.196"
 serde-wasm-bindgen = "0.6.3"
 serde_json = "1.0.113"
 strum = { version = "0.26.1", features = ["derive"] }
-time = { version = "0.3.34", features = [
```

Comment:
And this is the version on `main`, which has no issue getting a fixed `time` crate - the problem is really the `=0.3.31` in the v0.5.0 release